sr_robot_lib/src/generic_tactiles.cpp

```diff
@@ -88,7 +85,7 @@ namespace tactiles
               tactiles_vector->at(id_sensor).which_sensor =
                       static_cast<unsigned int>(static_cast<int16u>(status_data->tactile[id_sensor].word[0]));
             }
-            ROS_DEBUG_STREAM(" tact[" << id_sensor << "] = " << tactiles_vector->at(id_sensor).which_sensor);
+            ROS_INFO_STREAM(" tact[" << id_sensor << "] = " << tactiles_vector->at(id_sensor).which_sensor);
```

Comment:
Yes, that's a good point, I forgot to revert this change at some point. Although I'll be honest, personally I like being shown this information when I launch the hand, since it allows me very quickly to understand which sensors have been detected from the ones with issues

Reply:
Changed here: https://github.com/shadow-robot/sr-ros-interface-ethercat/pull/265/commits/a24b4bc1e46b8829861c4f9a3504d055279ece6e